packages/react-cosmos-router-proxy/src/index.js

```diff
@@ -31,7 +59,8 @@ export default () => {
   RouterProxy.propTypes = {
     ...proxyPropTypes,
     route: string,
-    url: string
+    url: string,
+    locationState: any
```

Comment:
Shouldn't this be an object type?

Reply:
Whilst `location.state` would typically be an object, I think it can actually be anything. There's nothing in the official docs, but it works perfectly fine with e.g. a string. The flow-typed libdefs also indicate any:
https://github.com/flowtype/flow-typed/blob/master/definitions/npm/react-router_v4.x.x/flow_v0.53.x-/react-router_v4.x.x.js#L9